src/lua/utils.h

```diff
@@ -388,9 +388,11 @@ luaL_toint64(struct lua_State *L, int idx);
 LUA_API int
 luaT_call(lua_State *L, int nargs, int nreturns);
 
-/*
+/**
  * Like luaL_dostring(), but in case of error sets fiber diag instead
  * of putting error on stack.
+ * @param L is a Lua state
+ * @param str
```

Comment:
Nits:

* Please, separate parameters from the brief description with an empty line.
* Isn't `is` redundant? AFAIS, we usually omit it in `@param`.
* How about some `str` description? 'Lua code to load and run' or kinda.

Reply:
> Please, separate parameters from the brief description with an empty line.

Done

> Isn't is redundant? AFAIS, we usually omit it in @param.

removed for consistency with other comments

> How about some str description? 'Lua code to load and run' or kinda.

```diff
@@ -391,8 +391,9 @@ luaT_call(lua_State *L, int nargs, int nreturns);
 /**
  * Like luaL_dostring(), but in case of error sets fiber diag instead
  * of putting error on stack.
- * @param L is a Lua state
- * @param str
+ *
+ * @param L Lua state
+ * @param str string with Lua code to load and run
  */
 int
 luaT_dostring(struct lua_State *L, const char *str);
```